masu/database/sql/reporting_ocpstoragelineitem_daily.sql

```diff
@@ -177,6 +177,8 @@ CREATE TEMPORARY TABLE reporting_ocpstoragelineitem_daily_{uuid} AS (
             ON li.report_id = ur.id
         JOIN reporting_ocpusagereportperiod AS rp
             ON li.report_period_id = rp.id
+        JOIN volume_nodes_{uuid} as uli
```

Comment:
I think we'll drop data if we do a JOIN here. The fact that we did have NULL values making it through means whatever records had a NULL will no longer be in the table. 

I took a single row from the OCP daily storage table that had a null for node and tried to find any record in the raw pod usage table that matched the pod name and there was nothing. That is troubling to me as far as data coming in. But because we don't get node from the volume data and we're assigning it after the fact I think we have to let the nulls through or risk losing usage data for storage.

Reply:
Yeah lets look into this a little deeper before merging anything.